test/weakmapMemoize.spec.ts

```diff
@@ -214,3 +170,52 @@ describe('Basic selector behavior with weakMapMemoize', () => {
     expect(called).toBe(2)
   })
 })
+
+const isCoverage = process.env.COVERAGE
```

Comment:
:hand: Just for consistency, can we leave the perf test changes out of this PR?  (Also, is this duplicate of what's in #640 ?)